packages/edit-site/src/components/sidebar-navigation-screen/style.scss

```diff
@@ -32,7 +32,18 @@
 .edit-site-sidebar-navigation-screen__back {
 	color: $gray-200;
 
+	// Focus (resets default button focus and use focus-visible).
+	&:focus:not(:disabled) {
+		box-shadow: none;
+		outline: none;
+	}
+	&:focus-visible:not(:disabled) {
```

Comment:
I wonder at what point we could consider this globally for all buttons and not just this override. cc @ciampo

Reply:
That would be a nice improvement, and in general I'd be up for bringing it to the source component.

The only challenge would be to make sure that there are no regressions when introducing the change. `Button` is a very complex component and it's not easy to grasp the extend of the consequences when making a change.

Would you be able to open a separate issue about this?